### PR TITLE
docs: use consistent capitalization, singular form of use

### PR DESCRIPTION
**Pull request by sublimator**

### CODE REVIEW

**Comment by sublimator** [comment]
bump ;)

**Comment by alandefreitas** [comment]
Looks great. Thanks.

**Comment by alandefreitas** [comment]
By the way, isn't this content replicated in the docs pages? How does it look there?

Also, could you rephrase the commit message to start with a noun? Something like "docs: comparison uses consistent capitalization"?

**Comment by sublimator** [comment]
Feel free to make any changes!

**Comment by vinniefalco** [comment]
as Nicholas is an infrequent contributor we can adjust the commit message for him or you can just merge it as is
